README.md

```diff
@@ -33,35 +33,80 @@ You can see the basic dummy app here:
   * [Dummy App](http://ember-form-master-2000.emersonlackey.com)
   * [Tests](http://ember-form-master-2000.emersonlackey.com/tests/)
 
-### Basic API
+### Overview
+
+Ember Form Master 2000 is an extensible library for creating forms in
+Ember.  The library comes out of the box with the standard form components
+you would expect and it is easy to add more custom components as
+needed.
+
+FM2000's primary entry point is the `fm-field` component.  This is
+used to render a form field to the page.  Even when you are making custom
+components you will still use `fm-field` to render them.
+
+Every time a field is rendered, it renders a `display` and a `widget`.
+`displays` are responible for the layout and css formatting of your form
+field.  For example, a using different `display`s you could opt to
+render a field using horizontal form style or a stacked form style.
+`widgets` are responsible for encapulsating the actual component
+that is manipulated.  There are built in widgets for things like
+`input` and `textarea` but you could add a custom date picker widget or
+a fancy autocomplete widget.  To learn more about creating custom `displays` 
+and `widgets` see the **Extending Form Master** below.
+
+#### A Brief Not on What FM2000 is Not
+
+FM2000 is all about rendering interactive forms in a standardized
+way with a concise and extensible syntax.  FM2000 is *not* about doing
+form validation or managing CRUD logic.
```

Comment:
Not sure where we ended up with validation hooks, but could be worth mentioning here